### ISSUE TITLE
Broken OG image

### ISSUE THREAD

**Issue by samajammin** [ethereum/eth2.0-deposit#373]
Current open graph image is broken
![Image 2021-03-08 at 1 26 09 PM](https://user-images.githubusercontent.com/8097623/110384091-06fde000-8012-11eb-902e-aae6710bad5e.jpg)

Should be: https://launchpad.ethereum.org/static/media/eth2-leslie-rhino.243747b9.png

**Comment by samajammin** [comment]
Hmmm ya I see that same issue.

It does appear to work fine w/ Facebook debugger:
![Image 2021-03-16 at 7 30 38 AM](https://user-images.githubusercontent.com/8097623/111326174-90706c00-8629-11eb-9283-5f7aa9aba0d8.jpg)

Not sure why Twitter reports errors... it should fall back to Open Graph metadata 🤔  but we can add twitter-specific metadata just to be sure.

**Comment by hwwhww** [comment]
It seems that it still doesn't work with Twitter. 😭 

![image](https://user-images.githubusercontent.com/9263930/111344292-34542900-86b7-11eb-9228-a71c7e5f1815.png)

**Comment by samajammin** [comment]
Hmmm so this is with https://github.com/ethereum/eth2.0-deposit/pull/381 deployed, ya?

**Comment by hwwhww** [comment]
@samajammin
yes, that was after Pyrmont release on March 17th: https://github.com/ethereum/eth2.0-deposit/pull/382

**Comment by samajammin** [comment]
Ultimately I suspect this is an issue with the fact that the site is not server-side rendered. We need dynamic tags (so that every page doesn't have the same title tag / meta description) but in doing so, the tags are not in the static HTML. I suspect Twitter's crawlers don't run the the JavaScript on the site; they simply take the meta tags from the initial bundle. 

Might require a more intensive fix than I hoped 😕  I was hoping Twitter's crawlers were smarter.

**Comment by samajammin** [comment]
More on this:
- https://stackoverflow.com/questions/59222217/twitter-card-meta-tags-work-in-index-html-not-in-react-helmet
- https://dev.to/zorfling/can-i-use-react-helmet-to-add-social-card-meta-tags-for-twitter-and-facebook-4a36
